webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { ComponentFixture, fakeAsync, flush, TestBed, tick } from '@angular/core/testing';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { TranslateFakeLoader, TranslateLoader, TranslateModule } from '@ngx-translate/core';
+import { expect } from 'chai';
+import * as moment from 'moment';
+import sinon from 'sinon';
+
+import { ChangesService } from '@mm-services/changes.service';
+import { ContactTypesService } from '@mm-services/contact-types.service';
+import { RulesEngineService } from '@mm-services/rules-engine.service';
+import { TasksActions } from '@mm-actions/tasks';
+import { TelemetryService } from '@mm-services/telemetry.service';
+import { TourService } from '@mm-services/tour.service';
+import { TasksComponent } from '@mm-modules/tasks/tasks.component';
+import { NavigationComponent } from '@mm-components/navigation/navigation.component';
+import { Selectors } from '@mm-selectors/index';
+
+describe('TasksComponent', () => {
+  let getComponent;
+  let Changes;
+  let RulesEngine;
```

Comment:
Some are starting lower case and others uppercase, maybe we can keep the original full name in camelCase:
Tour -> tourService

Reply:
I know, I preferred to keep the "old" variable names to keep the # of line changes lower, I think I didn't succeed anyway :D

Reply:
Changed.